src/main/resources/assets/not_enough_gamerules/lang/en_us.json

```diff
@@ -24,5 +24,7 @@
   "gamerule.skyHigh": "SkyHigh",
   "gamerule.skyHigh.description": "Defines the minimal high the player has to be before taking damages",
   "gamerule.tntExplodes": "TNT explosions",
+  "gamerule.keepEffects": "Keep effects",
+  "gamerule.keepEffects.description": "Keeps player's effects after death",
```

Comment:
Here too it must be sorted in alphabetical order :)

Reply:
:kappa: